Cubical/Foundations/Transport.agda

```diff
@@ -96,6 +96,12 @@ Iso.inv (pathToIso x) = transport⁻ x
 Iso.rightInv (pathToIso x) = transportTransport⁻ x
 Iso.leftInv (pathToIso x) = transport⁻Transport x
 
+substIso : ∀ {ℓ ℓ'} {A : Type ℓ} (B : A → Type ℓ') {x y : A} (p : x ≡ y) → Iso (B x) (B y)
+substIso B p = pathToIso (cong B p)
+
+substEquiv' : ∀ {ℓ ℓ'} {A : Type ℓ} (B : A → Type ℓ') {x y : A} (p : x ≡ y) → B x ≃ B y
+substEquiv' B p = isoToEquiv (substIso B p)
```

Comment:
This seems like a roundabout way to define this? Can't you use the one without the prime that's already in the library?

Reply:
I don't remember why I need this but I can imagine a it being more useful than the one already in the lib. The one in the lib doesn't have an explicitly defined inverse (it's an isEquiv proof by J) which I want to remember was causing issues. But maybe I could add a comment?

Reply:
A comment would be good

Reply:
I seem to recall that we had different proofs of this before...

Reply:
Couldn't move it next to it due to it needing pathToIso, but I added a comment.